tests/components/mqtt_room/test_sensor.py

```diff
@@ -47,6 +52,36 @@ async def assert_distance(hass, distance):
     assert state.attributes.get("distance") == distance
 
 
+async def assert_extra_room(hass, room, distance):
```

Comment:
I'm not sure this helps to keep the tests readable. May be we should not split this up in a function. You can use inner functions if needed.

Reply:
Sure, I can move it into the scope of `test_room_update_with_extra_rooms`, if that's what you mean.

Reply:
Thnx